komodod/amount

```diff
@@ -0,0 +1 @@
+amount=2000
```

Comment:
May I ask you to completely remove this file and pass it as ENV variable? In case we need to change it, we would need to rebuild image.